circle.yml

```diff
@@ -9,9 +9,16 @@ test:
     - PATH=$PATH:~/google_appengine ./smoke_test.py
 
 deployment:
-  appengine:
-    branch: /.*/
+  development:
+    branch: /^(?!master)$/
     commands:
       - rm -Rf venv/
-      - ~/google_appengine/appcfg.py update . --version=$(git rev-parse --abbrev-ref HEAD) --oauth2_refresh_token=$OAUTH2_REFRESH_TOKEN
+      - ~/google_appengine/appcfg.py update . -A caravel-code-reviews --version=$(git rev-parse --abbrev-ref HEAD) --oauth2_refresh_token=$OAUTH2_REFRESH_TOKEN
+      - curl https://$(git rev-parse --abbrev-ref HEAD)-dot-hosted-caravel.appspot.com >/dev/null
```

Comment:
no
and, well, it's a little dumb. Basically, it's supposed to catch 500 when we deploy a branch — but curl doesn't work that way. I should have removed it for this pull request :embarrassed: